src/app/profiles/store/reducers/categories.reducer.ts

```diff
@@ -0,0 +1,40 @@
+import { combineReducers } from 'redux';
+import {
+  fetched,
+  FetchedState,
+  fetching,
+  FetchingState,
+  lastFetched,
+  LastFetchedState
+} from 'app/store/collection/reducers';
+import { Categories } from 'app/lmem/category';
+import actionTypes, {
+  FETCH_CATEGORIES_SUCCESS,
+  FetchCategoriesSuccessAction
+} from 'app/actions/categories';
+
+export interface CategoriesCollection {
+  fetched: FetchedState;
+  fetching: FetchingState;
+  items: Categories;
```

Comment:
Why do we use an indexed collection, instead of a simple list, and thus we can’t use the generic`items` ?

Reply:
Because at the moment the list of categories is an enum on the backend side, so here it's a map and not a list, see :
https://staging-notices.bulles.fr/api/v3/categories

I'm open to discuss this format if you think it's not ideal.

Reply:
ping @JalilArfaoui

Reply:
I meant that we could store them in Redux side as a list of `{ id: string; label: string; }` for instance, and thus use the generic `items` reducer …

No big deal here …

Reply:
> we could store them in Redux side as a list of { id: string; label: string; }
Agreed, this the way we store other stuff, and we might move to this structure once the category is more than a map.
But in fact I do like the fact that this is a map here.

Anyway, does it mean you are approving this?

Reply:
Sorry @JalilArfaoui but since this is a blocker for other features, I'm merging this.

Reply:
Yep, sorry, I did not see you responses … I’m fine with this !